### ISSUE TITLE
Installation fails

### ISSUE THREAD

**Issue by seebi** [AKSW/QuitDiff#12]
Just wanted to diff some rdf files but installation failed with this error:

```
∴ pipx install QuitDiff
Fatal error from pip prevented installation. Full pip output in file:
    /Users/seebi/.local/pipx/logs/cmd_2024-07-08_22.13.38_pip_errors.log

pip seemed to fail to build package:
    rdflib==4.2.1

Some possibly relevant errors from pip install:
    error: subprocess-exited-with-error
    AttributeError: module 'distutils.util' has no attribute 'run_2to3'
    error: metadata-generation-failed

Error installing quitdiff.
```

**Comment by seebi** [comment]
update: https://github.com/AKSW/QuitDiff/tree/feature/poetry works fine (after updating the lock file)

**Comment by white-gecko** [comment]
Resolved by #11 .
